prov/efa/src/efa_hmem.h

```diff
@@ -30,6 +30,7 @@
  * SOFTWARE.
  */
 
+#include "efa_base_ep.h"
```

Comment:
I don't think this is needed here either

Reply:
fixed